app/notifications/process_notifications.py

```diff
@@ -267,14 +267,14 @@ def send_notification_to_queue(notification, research_mode, queue=None):
         if notification.sends_with_custom_number():
             deliver_task = provider_tasks.deliver_throttled_sms
             queue = QueueNames.SEND_THROTTLED_SMS
-        if not queue or queue == QueueNames.NORMAL:
+        if not queue:
```

Comment:
This looks like a fallback that was used when the sms/email specific queues were released. It is no longer possible to have `QueueNames.NORMAL` as far as I can tell.

Reply:
Let's test the 💩 out of this before releasing with some bug bash?

Reply:
Sounds good! I will wait until Monday to merge this.